lib/user_interface.rb

```diff
@@ -0,0 +1,20 @@
+class UserInterface
+  def initialize(board_formatter:, displayer:, input:)
+    @board_formatter = board_formatter
+    @displayer = displayer
+    @input = input
+  end
+
+  def display_board(rows)
+    formatted_board = board_formatter.format_board(rows)
```

Comment:
This line is pretty verbose! 3 mentions of `board` and three mentions of `board`!

Not a blocker, but maybe something to put on the list for later.